cogs5e/pbpUtils.py

```diff
@@ -58,9 +59,10 @@ async def embed(self, ctx, *, args):
         embed.set_image(url=args.last('image', '') if 'http' in str(args.last('image')) else '')
         embed.set_footer(text=args.last('footer', ''))
         try:
-            embed.colour = int(args.last('color', "0").strip('#'), base=16)
-        except:
-            pass
+            color = args.last('color')
+            embed.colour = int(color.strip('#'), base=16)
+        except (AttributeError, ValueError):
+            embed.colour = randint(0, 0xffffff)
```

Comment:
```suggestion
            pass
```
The color is already initalized to random in the constructor, this just keeps the import cleaner, and if we ever decide to change the default color in the future for any reason, it's one less refactor